terraform-infra-approvals/ccfr-fees-register-app.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources": [
+    {"type": "azurerm_template_deployment"}
```

Comment:
`azurerm_template_deployment` would give permission to create any Azure resources. Should seek SecOps approval to use this resource.

I'm happy that the use in this case is sensible - it was required to deploy Azure API management config before native Terraform resources were available. Native Terraform resources are available now and this should be refactored to use those in a sensible timescale but in the meantime this could be approved with IA approval. https://www.terraform.io/docs/providers/azurerm/r/api_management_api.html

Reply:
Agreed with @njrich28 and we are going to create a tech debt ticket to update this using native terraform resources in the next few sprints. However, this is currently blocking our build to go green and we need to move our changes into master branch asap.